packaging/greenboot/microshift-greenboot-required.sh

```diff
@@ -0,0 +1,209 @@
+#!/bin/bash
+set -e
+
+SCRIPT_NAME=$(basename $0)
+SCRIPT_LOG_FILE=/var/log/microshift-greenboot.log
+SCRIPT_PID=$$
+OCGET_OPT="--kubeconfig /var/lib/microshift/resources/kubeadmin/kubeconfig --no-headers"
+PODS_NS_LIST=(openshift-ovn-kubernetes openshift-service-ca openshift-ingress openshift-dns openshift-storage)
+PODS_CT_LIST=(2                        1                    1                 2             2)
+
+# Source Greenboot configuration file if it exists
+GREENBOOT_CONF_FILE=/etc/greenboot/greenboot.conf
+[ -f "${GREENBOOT_CONF_FILE}" ] && source ${GREENBOOT_CONF_FILE}
+WAIT_TIMEOUT_SECS_BASE=${MICROSHIFT_WAIT_TIMEOUT_SEC:-300}
+
+# Check the current user before the output redirect
+if [ $(id -u) -ne 0 ] ; then
+    echo "The '${SCRIPT_NAME}' script must be run with the 'root' user privileges"
+    exit 0
+fi
+
+# Redirect all the output to the log file and set the exit handler
```

Comment:
The output is redirected to system logs by Greenboot. 
I removed the script redirection to avoid custom log file handling in the future - great point.